database/migrations/2022_12_14_231443_add_merchandise_distributable_field.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('merchandise', static function (Blueprint $table): void {
+            $table->boolean('distributable')->default(false);
```

Comment:
```suggestion
            $table->boolean('distributable')->default(true);
```
Per Slack disucssion